### ISSUE TITLE
`prefer-set-has`: Add a `min` array length option

### ISSUE THREAD

**Issue by omril1** [sindresorhus/eslint-plugin-unicorn#2491]
### Description

Add a `min` option so only arrays from a certain known size will be linted (default 0).

The rule can sometimes be too strict, using sets over arrays is usually more performant the more elements it has, and it's negligible in small numbers.

### Fail

`min: 5`
```js
const array = [1, 2, 3, 4, 5, 6, 7];
const hasValue = value => array.includes(value);
```

### Pass

`min: 5`
```js
const array = [1, 2];
const hasValue = value => array.includes(value);
```

**Comment by sindresorhus** [comment]
Accepted.

The option should be called `minimumItems`.
